src/components/panel/panel.js

```diff
@@ -1306,10 +1386,22 @@ MdPanelRef.prototype._updatePosition = function(init) {
       this.panelContainer.addClass(MD_PANEL_HIDDEN);
     }
 
-    this.panelEl.css(MdPanelPosition.absPosition.TOP, positionConfig.getTop());
-    this.panelEl.css(MdPanelPosition.absPosition.BOTTOM, positionConfig.getBottom());
-    this.panelEl.css(MdPanelPosition.absPosition.LEFT, positionConfig.getLeft());
-    this.panelEl.css(MdPanelPosition.absPosition.RIGHT, positionConfig.getRight());
+    this.panelEl.css(
```

Comment:
Please don't use tabs to line up the second variables. It's one of the patterns I hate and personally, I think it makes it a lot less readable.